source/draw.c

```diff
@@ -720,14 +718,14 @@ void draw_interface(Entry_List_s* list, Instructions_s instructions)
             C2D_DrawSpriteTinted(&sprite_installed, &tint);
         }
 
-        if(!current_entry->placeholder_color)
+        if(current_entry->placeholder_color != 0)
         {
-            C2D_Image * image = NULL;
+            C2D_Image image;
             if(list->entries_count > list->entries_loaded*ICONS_OFFSET_AMOUNT)
-                image = list->icons[ICONS_VISIBLE*list->entries_loaded + (i - list->scroll)];
+                image = get_icon_at(list, ICONS_VISIBLE*list->entries_loaded + (i - list->scroll));
```

Comment:
readability: local variable for index; possibly break it down into parts e.g. `current := ICONS_VISIBLE * list->entries_loaded; offset := i - list->scroll`; also, not necessarily part of this PR but line 724 is missing spacing around `*`